pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
```

Comment:
Defining a `metrics` rest resource seems a bit odd considering we have images, pools, filesystem, block etc as resources. Would it make more sense if we have routes such as `/images/{image_id}/metrics`, /pools{pool}/metrics, etc? instead? API is our binding contract with the users so it would be hard to change later.